### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFConcepts_AnimatePathSample_ScreenReader: Screen reader focus is not going on the text under the first tab  

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1464]
Copied from VSO [952376](https://devdiv.visualstudio.com/DevDiv/_queries/edit/952376/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:

Application Name: .NET Core 3.0 WPF

.NET Core SDK version:  3.0.100-preview8-013442
Windows Version: Windows10

Screen Reader Vesion : Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

Repro Steps:

1) Run exe file for "Concepts" from the build app folder provided. 
2) Main Window for "Concepts" solution will appear. 
3) Select "Samples" tab item. 
4) From right navigation under "Animation" select "Animation Path Sample" 
 
 Actual:
 Screen reader focus is not going on the text which is under "Animation Path Sample" tab .
Similar issue is found under preview sample tab.

Same issue is observed through out the application where ever the XML tab is available.

Expected:

Screen reader focus should go on this text and screen reader should read this text properly.

User Impact:

Screen reader user will not get any information inside "Animation path sample"

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1:

https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by grubioe** [comment]
@stevenbrix - can you look into this one ?

**Comment by stevenbrix** [comment]
@grubioe sure

**Comment by stevenbrix** [comment]
@grubioe can you provide more detailed repro steps? I think I'm missing something as I'm not seeing a "build app folder"

**Comment by arpitmathur** [comment]
I'll link you to the folder @stevenbrix

**Comment by stevenbrix** [comment]
@grubioe I'm not sure there is an issue here, this looks like a bug with the sample. When you click on the hyperlink the content in the frame isn't given keyboard focus, so narrator doesn't start reading the elements. However, it is possible to tab to the element and get focus, and once that happens narrator will read the content correctly. If I make updates to the sample to properly set focus when the page is loaded, then it works as expected (according to this bug).

**Comment by grubioe** [comment]
Thanks for looking into it Steven.  I'll be closing then this issue here, I've added a tag in the internal bug related to cause being sample app.
